terraform/network-policies.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  configure_prometheus_network_policy = var.prometheus_app == null ? 0 : 1
+}
+
+data "cloudfoundry_app" "flt_web_app" {
+  depends_on = [cloudfoundry_app.app]
+  name_or_id = cloudfoundry_app.app.name
+  space      = data.cloudfoundry_space.space.id
+}
+
+data "cloudfoundry_app" "prometheus_app" {
+  count      = local.configure_prometheus_network_policy
+  name_or_id = var.prometheus_app
+  space      = data.cloudfoundry_space.space.id
+}
+
+resource "cloudfoundry_network_policy" "apply_prometheus_policy" {
```

Comment:
Name of the prometheus policy resource could be flt_prometheus_policy. I think this could mean apply for teacher training?

Reply:
Or even "prometheus_to_flt_policy" ?